docs/docs/manual/wikibase/configuration.md

```diff
@@ -73,6 +72,9 @@ Here is the manifest of Wikimedia Commons:
   },
   "editgroups": {
     "url_schema": "([[:toollabs:editgroups-commons/b/OR/${batch_id}|details]])"
+  },
+    "reconciliation": {
+    "endpoint": "https://commonsreconcile.toolforge.org/${lang}/api"
```

Comment:
This should not be needed: the reconciliation endpoints are now configured on a per-entity type basis, in the section above.

Reply:
OpenRefine gives you an error if its missing.

Reply:
ah yes, that is because this manifest is not compatible with version 3.5.2 :)

Reply:
Well 3.5.2. refers to this repository for manifests and it would fix the error

Reply:
Maybe it would let you add the manifest but then you would likely run into problems when you try to make edits, because OpenRefine 3.5 does not support editing on Wikibase instances which have federation enabled… it could potentially work for very restricted sorts of edits though.

I do agree that we should display a clearer error message, such as "This manifest version is not supported by OpenRefine 3.5.2", or something like that!